google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/internal/BufferingPullSubscriber.java

```diff
@@ -112,6 +112,29 @@ private synchronized void addMessages(Collection<SequencedMessage> new_messages)
     return ImmutableList.copyOf(collection);
   }
 
+  @Override
```

Comment:
Can't you implement these on the user side, by just calling pull() then testing for emptiness (hasNext, although this is not exactly what spark wants for hasNext I don't think).